### PR TITLE
[Actually Ready] TNO-2194: Enable drag and drop for content list

### PR DESCRIPTION
**Pull request by jdtoombs**
Made a conditional render to allow items to be drag and dropped in specific scenarios (ex. Manage Folder Content)

Simply passing a handleDrop function to `ContentList` will enable this feature; however, the list will no longer be sorted.

- also styling fixes for search page

image

### CODE REVIEW

File: app/subscriber/src/components/content-list/ContentList.tsx [hunk 1]
```diff
@@ -65,104 +64,80 @@ export const ContentList: React.FC<IContentListProps> = ({
...
-                      !!item.fileReferences.length &&
-                      !activeStream?.source &&
-                      !item.fileReferences[0].contentType.includes('image')
+      <Show visible={!handleDrop}>
+        {Object.keys(grouped).map((group) => (
+          <div key={group}>
+            <h2 className="group-title">{group}</h2>
+            <div>
+              {grouped[group].map((item) => (
+                <ContentRow
```
Comment (jdtoombs) [inline app/subscriber/src/components/content-list/ContentList.tsx:73]: Move bulk of elements to `ContentRow` for a cleaner conditional render when determining if draggable or not

File: app/subscriber/src/components/content-list/ContentRow.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
+import moment from 'moment';
```
Comment (jdtoombs) [inline app/subscriber/src/components/content-list/ContentRow.tsx:1]: Same code that was in `ContentList` but abstracted

File: app/subscriber/src/components/content-list/styled/ContentRow.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
+import styled from 'styled-components';
```
Comment (jdtoombs) [inline app/subscriber/src/components/content-list/styled/ContentRow.tsx:1]: Also some styling fixes in here for the updated search view

**Review by Fosol — APPROVED** [review]

File: app/subscriber/src/components/content-list/ContentList.tsx [hunk 4]
```diff
@@ -35,7 +35,7 @@ export const ContentList: React.FC<IContentListProps> = ({
 }) => {
   const navigate = useNavigate();
   const { groupBy, setActiveStream, activeFileReference } = React.useContext(ContentListContext);
-  const grouped = groupContent(groupBy, content);
+  const grouped = groupContent(groupBy, [...content]);
```
Comment (jdtoombs) [inline app/subscriber/src/components/content-list/ContentList.tsx:38]: this costed me a few hours

**Comment by jdtoombs** [comment]
Just trying to find the best way to fix this 
image

**Review by Fosol — APPROVED** [review]

**Pull request merged**